jupyter/trustyai/ubi8-python-3.8/Pipfile

```
@@ -7,42 +7,47 @@ name = "pypi"
 
 [packages]
 # TrustyAI packages
-trustyai = "~=0.3.0"
+trustyai = "~=0.5.0"
```

```suggestion
trustyai = "~=0.6.0"
```

There is a new version for TrustyAI